.github/workflows/ci-pullrequest-pipeline.yml

```
@@ -0,0 +1,29 @@
+name: Web Frontend CI Pipeline on PULL REQUEST
+on: [pull_request]
+
+jobs:
+  build-and-test-job:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [18.x]
+
+    steps:
+      - name: Checkout ✅
+        uses: actions/checkout@v3
+
+      - name: Setup 🏗
+        uses: actions/setup-node@v3
+
+      - name: Install ⚙️
+        run: npm ci
+
+      - name: Build
+        run: |
+          npm run build
+      - name: Lint
+        run: |
+          npm run lint
+      - name: Test
+        run: |
+          npm run test
```

```suggestion
      - name: Lint
        run:  npm run lint
```

The other workflow already executes the tests. Not needed to run them twice.

---

True, done